content/en/tracing/profiler/enabling/dotnet.md

```diff
@@ -20,49 +20,109 @@ further_reading:
 Datadog .NET Profiler is currently in public beta. Datadog recommends evaluating the profiler in a non-sensitive environment before deploying in production.
 </div>
 
-<br>
+The following profiling features are available:
 
-## Requirements
+|      Feature         | Set this environment variable to 1   |
+|----------------------|--------------------------------------|
+| Wall time profiling  | Always enabled                       |
+| CPU time profiling   | DD_PROFILING_CPU_ENABLED             |
```

Comment:
I would not explain _how_ to enable it in this table, but the supported platforms (ex: Wall Time on .NET Framework 4.6.1+ and .NET Core 3.1+). I would then explain how to enable the CPU and Exception profiler in a dedicated section under, like for [Java's Allocation profiler](https://docs-staging.datadoghq.com/chrisnas/dotnet-beta-linux/tracing/profiler/enabling/java?tab=commandarguments#enabling-the-allocation-profiler). That's mostly because we expect to have _all_ profilers enabled by default at some point (on relevant platforms), and only have them opt-out.

Reply:
Good point: I will also document these environment variables in the Configuration section.
Since all profiling features are available on all platforms and for all .NET versions, it would make no sense to have a table.
I've been told that profilers features are DISABLED by default (BTW, the .NET profiler as a whole is disabled by default)

Reply:
It's not my understanding that at the moment all features we are planning (Allocation profiling for example) will be available on all platforms right away. We'll strive to have them on all platforms, but that will take time, correct? If so, then we can update the documentation as we go.

Reply:
Also, on the disabled-by-default, we try to enable as much as we can by default, and only disable the ones we have no good support for in the runtime, leading to high overhead. As soon as we are confident we have low overhead, we have it enabled by default.

For example, Allocation profiling in Java is enabled by default with Java 16+ because we know there is support for a performant allocation sampling event. In lower versions, it's disabled by default.

Reply:
The table has been removed and replaced by a short bullets list